docs/docs/icicle/golang-bindings/ntt.md

```diff
@@ -64,6 +64,7 @@ type NTTConfig[T any] struct {
     Ctx cr.DeviceContext
     CosetGen T
     BatchSize int32
```

Comment:
I guess `NttAlgorithm` also missing from this config? @yshekel 
More generally, this level of documentation should really be generated from doc comments imo

Reply:
Isn't it the same one?

Reply:
Yea you also pointed this out, it's a duplicate

Reply:
yeah it should be... just its very hard to integrate ill write why in slack.